requirements/dev.txt

```diff
@@ -17,4 +17,5 @@ torch>=1.13.1
 httpx==0.27.0
 ruff==0.4.7
 aioresponses==0.7.6
-factory-boy==3.3.0
\ No newline at end of file
+factory-boy==3.3.0
+tenacity==8.3.0
```

Comment:
Do we need this in prod.txt and dev.txt?

Reply:
no, we don't need it here. just in prod.